js/miniature.js

```diff
@@ -0,0 +1,19 @@
+import {getSimilarObjects} from './data.js';
+
+const fragment = document.createDocumentFragment('.pictures');
+const pictureTemplate = document.querySelector('#picture').content;
+
+
+const similarObjects = getSimilarObjects();
+for (let i = 0; i < similarObjects.length; i++) {
+  const element = pictureTemplate.cloneNode(true);
```

Comment:
```suggestion
  const picture = pictureTemplate.cloneNode(true);
```

https://up.htmlacademy.ru/javascript/28/criteries#d6